plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -1633,9 +1633,15 @@ export function stateModelFactory(pluginManager: PluginManager) {
       },
     }))
     .preProcessSnapshot(snap => {
+      if (!snap) {
+        return snap
+      }
       const { highlight, ...rest } = snap
       return {
-        highlight: highlight ? [...highlight] : [],
+        highlight:
+          Array.isArray(highlight) || highlight === undefined
+            ? highlight
+            : [...highlight],
```

Comment:
this tries to spread the old style of highlight (an object) which gives an error that it "is not iterable" so i think just [highlight] instead of [...highlight]

can test by loading a highlight on e.g. main and transferring to this branch